routingpy/expansion.py

```diff
@@ -128,7 +128,7 @@ def center(self) -> Optional[Union[List[float], Tuple[float]]]:
         """
         The center coordinate in [lon, lat] of the expansion, which is the location from the user input.
 
-        :rtype: list of float
+        :rtype: list of float or None
```

Comment:
actually these `rtype:` statements can be removed when it's properly typed, they're just a crutch

Reply:
no need to remove them now, it's a cleanup for another rainy day:) just saying, they don't need updating anymore